pkg/dashboard/pod.go

```diff
@@ -328,6 +331,71 @@ func (api *API) downloadDebugFile() gin.HandlerFunc {
 	}
 }
 
+func (api *API) downloadDebugInfo() gin.HandlerFunc {
+	return func(c *gin.Context) {
+		namespace := c.Param("namespace")
+		name := c.Param("name")
+		pod, err := api.client.GetPod(c, name, namespace)
+		if err != nil || pod == nil {
+			podLog.Error(err, "Failed to get pod")
+			return
+		}
+		if !utils.IsMountPod(pod) {
+			podLog.Error(err, "not a mount pod")
+			return
+		}
+		// create tmp dir
+		podTmpDir := fmt.Sprintf("/tmp/%s/%s", StripDir(namespace), StripDir(name))
+		dir := filepath.Join(podTmpDir, "info")
+		err = os.MkdirAll(dir, 0777)
+		if err != nil {
+			podLog.Error(err, "Failed to create tmp dir")
+			return
+		}
+
+		defer os.RemoveAll(dir)
+
+		// download csi node log
+		csiNodeLog := filepath.Join(dir, "csi-node.log")
+		csiNode, err := api.getCSINode(c, pod.Spec.NodeName)
+		if err != nil {
+			podLog.Error(err, "get csi node error", "node", pod.Spec.NodeName)
+			return
+		}
+		if err := resource.DownloadPodLog(c, api.client, csiNode.Namespace, csiNode.Name, "juicefs-plugin", csiNodeLog); err != nil {
+			podLog.Error(err, "Failed to download pod log")
+			return
+		}
+		// download csi node yaml
+		csiNodeYaml := filepath.Join(dir, "csi-node.yaml")
+		if err := DownloadPodYaml(csiNode, csiNodeYaml); err != nil {
+			podLog.Error(err, "Failed to download pod yaml")
+			return
+		}
+		// download mount pod log
+		mountPodLog := filepath.Join(dir, "mount-pod.log")
+		if err := resource.DownloadPodLog(c, api.client, namespace, name, common.MountContainerName, mountPodLog); err != nil {
+			podLog.Error(err, "Failed to download pod log")
+			return
+		}
+		// download mount pod yaml
+		mountPodYaml := filepath.Join(dir, "mount-pod.yaml")
+		if err := DownloadPodYaml(pod, mountPodYaml); err != nil {
+			podLog.Error(err, "Failed to download pod yaml")
+			return
+		}
+
```

Comment:
can we also collect

- pv/pvc yaml
- global config

Reply:
do we also need csi controller log/yaml ?